docs/references/kontena-yml.md

```diff
@@ -314,12 +349,12 @@ db:
 
 #### external_links
 
-Link to services in the same grid outside application scope. `external_links` follow semantics similar to links.
+Link to services in the same grid outside stack scope. `external_links` follow semantics similar to links. However you have to use `<stack>/<service>` format instead:
```

Comment:
The `stack/service` syntax is also used for `links`, so this isn't any different from `links`?